src/main/java/net/snowflake/ingest/streaming/internal/RegisterService.java

```diff
@@ -171,10 +174,9 @@ List<FlushService.BlobData<T>> registerBlobs(Map<String, Timer.Context> latencyT
                       e.getCause() == null ? null : e.getCause().getMessage(),
                       getStackTrace(e.getCause()));
               logger.logError(errorMessage);
-              if (this.owningClient.getTelemetryService() != null) {
-                this.owningClient
-                    .getTelemetryService()
-                    .reportClientFailure(this.getClass().getSimpleName(), errorMessage);
+              // TODO: @rcheng question - should i try another telemetryservice get here?
```

Comment:
I didn't see any additional telemetry client setup above this point in the method, so I'm assuming it is safe to move the getTelemetryClient() call to the beginning of the method?